crates/api_models/src/payouts.rs

```diff
@@ -205,110 +234,14 @@ pub struct Bank {
-            bank_city: String,
-        }
-
-        let p = BankParams::deserialize(deserializer)?;
-        match
-            (
-                p.bank_account_number.as_ref(),
-                p.bank_routing_number.as_ref(),
-                p.iban.as_ref(),
-                p.bic.as_ref(),
-                p.bank_sort_code.as_ref(),
-                p.blz.as_ref(),
-                p.bank_transit_number.as_ref(),
-            )
-        {
-            (None, None, None, None, None, None, None) =>
-                Err(
-                    de::Error::custom(
-                        "Invalid bank details, atleast one of bank_account_number + bank_routing_number / bic / bank_sort_code / blz / bank_transit_number OR iban must be provided."
-                    )
-                ),
-            (None, Some(_), _, _, _, _, _) =>
-                Err(
-                    de::Error::custom(
-                        "Invalid bank details, bank_account_number is required when passing bank_routing_number"
-                    )
-                ),
-            (None, _, _, _, Some(_), _, _) =>
-                Err(
-                    de::Error::custom(
-                        "Invalid bank details, bank_account_number is required when passing bank_sort_code"
-                    )
-                ),
-            (None, _, _, _, _, Some(_), _) =>
-                Err(
-                    de::Error::custom(
-                        "Invalid bank details, bank_account_number is required when passing blz"
-                    )
-                ),
-            (None, _, _, _, _, _, Some(_)) =>
-                Err(
-                    de::Error::custom(
-                        "Invalid bank details, bank_account_number is required when passing bank_transit_number"
-                    )
-                ),
-            (Some(_), None, _, None, None, None, None) =>
-                Err(
-                    de::Error::custom(
-                        "Invalid bank details, bank_account_number should be passed along with atleast one of bank_routing_number, bic, bank_sort_code, blz or bank_transit_number"
-                    )
-                ),
-            _ =>
-                Ok(Self {
-                    bank_account_number: p.bank_account_number,
-                    bank_routing_number: p.bank_routing_number,
-                    iban: p.iban,
-                    bic: p.bic,
-                    bank_sort_code: p.bank_sort_code,
-                    blz: p.blz,
-                    bank_transit_number: p.bank_transit_number,
-                    bank_name: p.bank_name,
-                    bank_country_code: p.bank_country_code,
-                    bank_city: p.bank_city,
-                }),
-        }
-    }
+    /// [8 / 11 digits] Bank Identifier Code (bic) / Swift Code - used in many countries for identifying a bank and it's branches
+    #[schema(value_type = String, example = "HSBCGB2LXXX")]
+    pub bic: Option<String>,
```

Comment:
mask it with secret

Reply:
Resolved in a separate commit - https://github.com/juspay/hyperswitch/pull/1384/commits/ac41daafceda596d06c428eb4b15eb15fadd337b